lib/OpenQA/Git.pm

```diff
@@ -54,7 +54,13 @@ sub set_to_latest_master ($self, $args = undef) {
     }
 
     if (my $update_branch = $self->config->{update_branch}) {
-        my $res = run_cmd_with_log_return_error([@git, 'rebase', $update_branch]);
+        my $res = run_cmd_with_log_return_error([@git, 'restore', '--staged', '.']);
+        return _format_git_error($res, 'Unable to restore staged files') unless $res->{status};
+        $res = run_cmd_with_log_return_error([@git, 'restore', '.']);
+        return _format_git_error($res, 'Unable to restore files') unless $res->{status};
```

Comment:
I suppose `git reset --hard HEAD` would be simpler than these two restore steps. I suppose then you could even avoid the cleanup.

Reply:
Yeah that seems like an easier solution, I'll make that change